src/EditorFeatures/Test/Diagnostics/DiagnosticTagSourceTests.cs

```diff
@@ -1,110 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Linq;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics;
-using Microsoft.CodeAnalysis.Editor.Shared.Tagging;
-using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Microsoft.CodeAnalysis.SolutionCrawler;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Tagging;
-using Roslyn.Test.Utilities;
-using Roslyn.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics
-{
-    public class DiagnosticTagSourceTests
-    {
-        [WpfFact, Trait(Traits.Feature, Traits.Features.Diagnostics)]
-        public void Test_TagSourceDiffer()
-        {
-            using (var workspace = CSharpWorkspaceFactory.CreateWorkspaceFromFiles(new string[] { "class A { }", "class E { }" }, CSharpParseOptions.Default))
```

Comment:
@heejaechang This pull request deleted files which existed on disk but were not included in any project. Jason is asking if you meant for this to be included in the build, because so far it was just a stray file.